docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: index-vincent-aceto-story
+title: "Hacktoberfest Stories: A Hacktoberfest Composition: Redis and Docker"
+sidebar_label: Vincent's Hacktoberfest at Redis Story
+slug: /hacktoberfest/stories/vincent-aceto
+---
+
+Hello!  My name's [Vincent Aceto](https://www.vincentaceto.com/) and I am a Software Engineer based in New York City.  Throughout the week, you can find me hacking away on mobile and TV applications over at Equinox Media.  On the weekends, when I'm not getting lost somewhere on my skateboard, I'll be nose deep in some open source or personal projects.
+
+October is a special month for those who enjoy working on exciting software projects.  In fact, there couldn’t be a more perfect month to get into software exploration and contribution; it's collectively known as Hacktoberfest!  Hacktoberfest is a community-led effort to encourage open source contributing and foster learning.  I am a huge advocate for open source, so getting involved in Hacktoberfest is such a joy; if I have the chance to learn something new or brush up on some skills, then definitely count me in.
+
+Now, rewind the clock a bit, and you'd find me perusing Github's Hacktoberfest-tagged issues.  I wanted to find the best first contribution for the month's coding festivities.  While searching, I had one very important criterion that the introductory issue needed to satisfy: to work with a technology that I do not use on a daily basis.  I wanted to make sure that I walked away with a newfound knowledge that would benefit my career.  After some time, my eyes landed on a Redis Developer Community's issue - I knew it immediately, this was perfect!  The checkbox was ticked, for I do not regularly work with Redis.  I was now ready to kick off the Hacktoberfest celebration.
```

Comment:
@SuzeShardlow suggest changing "landed on a Redis Developer Community's issue" to "landed on a Redis Developer Community isue"

Reply:
Fixed in 0eb43d22d